cmd/validators/validators.go

```diff
@@ -39,11 +40,16 @@ func Validate() cobra.PositionalArgs {
 }
 
 func CmdError(err error, commandPath string) error {
-	errorMessage := fmt.Sprintf(
-		"%s. See `%s --help` for supported flags and usage.",
-		err.Error(),
-		commandPath,
-	)
+	errorMessage := err.Error() + "."
+
+	// show additional help if a missing flag can be set in the config file
+	if strings.Contains(err.Error(), cliflags.AccessTokenFlag) {
```

Comment:
Right now we only check the access-token flag because the other ones have default values.